### PR TITLE
MySql-Cygnus Tutorial documentation

### PR DESCRIPTION
**Pull request by Joudsi**
@jmcanterafonseca

### CODE REVIEW

File: doc/storing-data-cygnus-mysql/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+        - mysql:iot-mysql
+    expose:
+        - "5050"
+    ports:
+        - "5050:5050"
+        - "8081:8081"
+    environment:
+        - CYGNUS_MYSQL_USER=root
+        - CYGNUS_MYSQL_PASS=mysql
+
```
Comment (jmcanterafonseca) [inline doc/storing-data-cygnus-mysql/docker-compose.yml]: blank line needed

File: doc/storing-data-cygnus-mysql/how-to-store-data-cygnus-mysql.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+c30a8ba06958        fiware/cygnus-ngsi:latest   "/cygnus-entrypoint.…"   11 minutes ago      Up 11 minutes       0.0.0.0:5050->5050/tcp, 0.0.0.0:8081->8081/tcp   cygnus
+187e9b6f9f49        mongo:3.4                   "docker-entrypoint.s…"   11 minutes ago      Up 11 minutes       27017/tcp                                        mongo
+59ed1e7daba8        mysql:latest                "docker-entrypoint.s…"   11 minutes ago      Up 11 minutes       3306/tcp                                         mysql
+```
+
+We see that four containers are running now. Here, we can notice that the Orion Context Broker is running on port 1026, Cygnus-ngsi exposes two ports (8081 and 5050), the API must be running in a port you configured and 8081 (management port) is the default one. Whereas port 5050 is the port that receives notifications from Orion.  For more info you can check the documentations [here](http://fiware-cygnus.readthedocs.io/en/1.7.1/cygnus-common/installation_and_administration_guide/management_interface_v1/index.html)
```
Comment (jmcanterafonseca) [inline doc/storing-data-cygnus-mysql/how-to-store-data-cygnus-mysql.md]: s/documentations/documentation

Cygnus-ngsi exposes two ports 8081, for management purposes, and 5050 for receiving notifications from Orion.

File: mkdocs.yml [hunk 3]
```diff
@@ -18,6 +18,7 @@ pages:
         - 'Subscription to context changes': 'development-context-aware-applications/how-to-subscribe-to-changes-on-context-information.md'
         - 'Geolocated context queries': 'development-context-aware-applications/how-to-perform-geo-located-queries.md'
       - 'Generate the history of Context Information using STH-Comet': 'generating-historical-context-information-sth-comet/how-to-generate-the-history-of-Context-Information-using-STH-Comet.md'
+      - 'Storing data in MySQL through Cygnus': 'storing-data-cygnus-mysql/how-to-store-data-cygnus-mysql.md'
```
Comment (jmcanterafonseca) [inline mkdocs.yml]: Store context information in MySQL through Cygnus

File: doc/storing-data-cygnus-mysql/how-to-store-data-cygnus-mysql.md [hunk 4]
```diff
@@ -0,0 +1,251 @@
+# How to store data in mySQL using Cygnus
```
Comment (flopezag) [inline doc/storing-data-cygnus-mysql/how-to-store-data-cygnus-mysql.md]: MySQL

File: doc/storing-data-cygnus-mysql/docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
```
Comment (jmcanterafonseca) [inline doc/storing-data-cygnus-mysql/docker-compose.yml]: only one blank line

File: doc/storing-data-cygnus-mysql/how-to-store-data-cygnus-mysql.md [hunk 6]
```diff
@@ -0,0 +1,251 @@
...
+7406940f3438        fiware/orion:latest         "/usr/bin/contextBro…"   11 minutes ago      Up 11 minutes       0.0.0.0:1026->1026/tcp                           orion
+c30a8ba06958        fiware/cygnus-ngsi:latest   "/cygnus-entrypoint.…"   11 minutes ago      Up 11 minutes       0.0.0.0:5050->5050/tcp, 0.0.0.0:8081->8081/tcp   cygnus
+187e9b6f9f49        mongo:3.4                   "docker-entrypoint.s…"   11 minutes ago      Up 11 minutes       27017/tcp                                        mongo
+59ed1e7daba8        mysql:latest                "docker-entrypoint.s…"   11 minutes ago      Up 11 minutes       3306/tcp                                         mysql
+```
+
+We see that four containers are running now. Here, we can notice that the Orion Context Broker is running on port 1026. Cygnus-ngsi exposes two ports for management purposes, and 5050 for receiving notifications from Orion. The API must be running on port you configured and 8081 (management port) is the default one. For more info you can check the documentations [here](http://fiware-cygnus.readthedocs.io/en/1.7.1/cygnus-common/installation_and_administration_guide/management_interface_v1/index.html)
```
Comment (jmcanterafonseca) [inline doc/storing-data-cygnus-mysql/how-to-store-data-cygnus-mysql.md]: We see that four containers are running now. Here, we can notice that the Orion Context Broker is running on port 1026. Cygnus-ngsi exposes two ports, 8081 for management purposes and 5050 for receiving notifications from Orion. For more info you can check the documentation [here](http://fiware-cygnus.readthedocs.io/en/1.7.1/cygnus-common/installation_and_administration_guide/management_interface_v1/index.html)

**Comment by jmcanterafonseca** [comment]
LGTM, landing

**Pull request merged**